gen/calc.py

```diff
@@ -575,6 +575,34 @@ def fz_err(msg):
             validate_int_in_range(port, 1, 65535)
 
 
+def calculate_fair_sharing_excluded_resource_names(scarce_resources):
+    return ','.join(validate_json_list(scarce_resources))
+
+
+def calculate_filter_gpu_resources(scarce_resources):
+    items = validate_json_list(scarce_resources)
+    if not len(items):
+        return 'false'
```

Comment:
This `if` is redundant, if the list is empty then we'll get to the `return 'false'` at the end of the function.